app/config/index.js

```diff
@@ -0,0 +1,86 @@
+const Joi = require('joi')
+const mqConfig = require('./messaging')
+
+// Define config schema
+const schema = Joi.object({
+  port: Joi.number().default(3000),
```

Comment:
Most of this config isn't used and is therefore redundant. I'd prefer not to add config that we aren't using, and to only add it when the features that use it are added.

From what I can tell we only need `cookiePassword`, `cookieOptions` and `googleTagManagerKey` in this PR, so I think the rest should be removed.